docs/overview/deep_dive/inplace_updates.rst

```diff
@@ -287,31 +287,35 @@ The implementations of :func:`ivy.tan` for each backend are as follows.
 
 .. code-block:: python
 
-    @_handle_0_dim_output
+    @_scalar_output_to_0d_array
     def tan(x: np.ndarray, /, *, out: Optional[np.ndarray] = None) -> np.ndarray:
         return np.tan(x, out=out)
 
+
     tan.support_native_out = True
 
 **TensorFlow** (no :code:`support_native_out` attribute):
 
 .. code-block:: python
 
     def tan(
-        x: Union[tf.Tensor, tf.Variable],
```

Comment:
Could you please revert the change to indentation for this code block?

Reply:
done